.gitignore

```diff
@@ -5,7 +5,7 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
-
+src/Verification_Team_Roster.xlsx
```

Comment:
This is not necessary. In addition, the actual file path is [here](https://github.com/GryphonRacingFSAE/Discord-Bot/blob/6c5cb59c45417faa52f9147ae5a39b5ab01c0cf2/src/vertification.ts#L51)